dist/gmusic.js

```diff
@@ -195,6 +195,47 @@ proto.playback = {
     this.playback._sliderEl.dispatchEvent(evt);
   },
 
+  // Return song duration in milliseconds
+  getSongDuration: function () {
+    return this.playback._sliderEl.max;
+  },
+
+  // PlayPause element states:
```

Comment:
This comment should be inside the function since it's not describing anything about `getPlaybackState`